code/controllers/subsystem/tremors.dm

```diff
@@ -0,0 +1,24 @@
+SUBSYSTEM_DEF(tremors)
```

Comment:
Is it not possible to use the weather subsystem for this?

Reply:
it would compete with ash storms, and i'd like them to be independent from each other

Reply:
With some refactoring, they could go together. The weather SS is already kinda underutilized; I feel like it would be less silly to integrate them instead of using an entirely new subsystem for a single weather event.